synfig-core/src/tool/optionsprocessor.cpp

```diff
@@ -684,6 +696,22 @@ Job SynfigCommandLineParser::extract_job()
 	    throw SynfigToolException(SYNFIGTOOL_MISSINGARGUMENT,
                                   _("No input file provided."));
 	}
+        
+	if(!set_renderer.empty())
+	{
+		auto renderers = rendering::Renderer::get_renderers();
+		auto ri = renderers.find(set_renderer);
+		if (ri == renderers.end() || !ri->second)
+		{
+			synfig::error("Invalid renderer: %s", set_renderer.c_str()); 
+			for(const auto& iter : synfig::rendering::Renderer::get_renderers()) {
+				std::cout << (iter.first).c_str() << " - " << iter.second->get_name() << std::endl;
+			}
```

Comment:
@ice0 do you think it should be std::cerr or even synfig::error? Or even list the valid renderers here on error?

Reply:
@rodolforg 
I think yes. If the user uses the wrong renderer, then his next step is to check the list of available visualizers, so we will just show them.
Also it looks like better to write it to stderr for consistency.

Reply:
@ice0 , should I replace all `std::cout` with `std::cerr` ?

Reply:
@veermetri05 
Yes, this will make the output more consistent.

Reply:
@veermetri05 not all, but those that are error report (like this one).
I only found one more (pointed in another comment below).

Reply:
@veermetri05 ping

Reply:
Done!